### PR TITLE
added lead field to shout, new table event

### PR DESCRIPTION
**Pull request by bniwredyc**

### CODE REVIEW

File: resolvers/__init__.py [hunk 1]
```diff
@@ -90,13 +90,6 @@
     "update_shout",
     "delete_shout",
     "markdown_body",
-    "load_drafts",
```
Comment (alexeyqu) [inline resolvers/__init__.py:93]: already deleted from code

File: resolvers/auth.py [hunk 2]
```diff
@@ -110,6 +110,7 @@ def generate_unique_slug(src):
 
 @mutation.field("registerUser")
 async def register_by_email(_, _info, email: str, password: str = "", name: str = ""):
+    email = email.lower()
```
Comment (alexeyqu) [inline resolvers/auth.py:113]: do you want to include these changes here?

File: schema.graphql [hunk 3]
```diff
@@ -417,6 +417,7 @@ type Shout {
   id: Int!
   slug: String!
   body: String!
+  lead: String
```
Comment (alexeyqu) [inline schema.graphql:420]: maybe `String!`? right now it can be `null` or empty `""`, is there a difference for the API?

File: orm/event.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+
+from sqlalchemy import Column, String, JSON, Boolean, ForeignKey, DateTime
+from base.orm import Base
+
+
+class Event(Base):
```
Comment (alexeyqu) [inline orm/event.py:7]: discussed via chat

**Review by alexeyqu — CHANGES_REQUESTED** [review]
reject because need to rename the "event" table

File: resolvers/auth.py (hunk 2 above)
Comment (bniwredyc) [reply resolvers/auth.py:113]: yes, i'm a lazy ass :)

**Review by alexeyqu — APPROVED** [review]

**Pull request merged**